tests/plugin_tests/transaction_status.cpp

```diff
@@ -206,7 +206,7 @@ BOOST_AUTO_TEST_CASE( transaction_status_test )
       BOOST_REQUIRE( api_return.block_num.valid() == false );
 
       api_return = tx_status_api->api->find_transaction( { .transaction_id = tx1.id(), tx1_expiration } );
-      BOOST_REQUIRE( api_return.status == expired_reversible );
+      BOOST_REQUIRE( api_return.status == unknown );
```

Comment:
I don't see any expired reversible tests now...

Reply:
There is expiration time related tests at the bottom, it covers `expired_irreversible`, `expired_reversible`, and `too_old`.